pkg/capacity/planner.go

```diff
@@ -574,6 +580,55 @@ func (p *NaivePlanner) handlePoolExpiration(cancelOld bool) error {
 	return nil
 }
 
+func (p *NaivePlanner) syncPools() error {
+	pools, err := types.GetPools(p.ctx, p.db)
+	if err != nil {
+		return err
+	}
+
+	for _, pool := range pools {
+		err := p.updateUsedCapacityPool(pool.ID)
+		if err != nil {
+			return err
+		}
+	}
+
+	return nil
+}
+
+func (p *NaivePlanner) updateUsedCapacityPool(poolID schema.ID) error {
```

Comment:
Why you pass the pool id and then do a get, while u already have the object in the returned pools at line 584. wouldn't it be more efficient if u pass a pointer instead another db operation